getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,73 @@
+# Instructions for macOS
+
+Install [Homebrew](http://brew.sh/) to give your mac a package manager. Open up your terminal emulator of choice and run:
+
+```shell
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
+```
+
+Install git:
+
+```shell
+$ brew install git
+```
+
+Install python and its package manager, pip:
+
+```shell
+$ brew install python
+$ curl -O http://python-distribute.org/distribute_setup.py
+$ python distribute_setup.py
+$ curl -O https://raw.github.com/pypa/pip/master/contrib/get-pip.py
+$ python get-pip.py
+$ rm distribute_setup.py get-pip.py
+```
```

Comment:
Same comment as above, `python` and `pip` are installed by default. Also, if we are using `brew` for other installations, why not run `brew install pip`?